test/dlc_tests/conftest.py

```diff
@@ -213,6 +217,15 @@ def pytest_configure(config):
     config.addinivalue_line(
         "markers", "canary(message): mark test to run as a part of canary tests."
     )
+    config.addinivalue_line(
+        "markers", "integration(ml_integration): mark what the test is testing."
+    )
+    config.addinivalue_line(
+        "markers", "model(model_name): The name of the pytest model being tested"
+    )
+    config.addinivalue_line(
+        "markers", "multinode(num_instances): The number of instances the test is run on"
```

Comment:
This is an optional marker. If it is not used, we assume single node

Reply:
no longer relevant -- we will write "multinode" if we find multinode in the function metadata